javascript/operators/optional_chaining.json

```diff
@@ -0,0 +1,60 @@
+{
+  "javascript": {
+    "operators": {
+      "optional_chaining": {
+        "__compat": {
+          "description": "Optional chaining operator (<code>?.</code>)",
```

Comment:
Is the description here too long to work well in the sidebar and such? Can we come up with something that's a little shorter? I don't know that we can, but it's worth considering.

Other than this issue, this LGTM.

Reply:
This is not used as a sidebar label and it is similar to what we have for the bitwise operators etc. in BCD. I'm happy to leave this as is.